__test__/api/admins/changePassword.test.js

```diff
@@ -0,0 +1,71 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+const AdminModel = require("../../../models/admins");
+const { getOrgToken } = require("../../../utils/auth/tokenGenerator");
+const { comparePassword } = require("../../../utils/auth/passwordUtils");
+
+const request = supertest(app);
+
+describe("PATCH /v1/admins/:adminId/change-password", () => {
+  let passwordChangeInfo = {
+    oldPassword: "password",
+    newPassword: "passcode",
+  };
+  let url = `/v1/admins/change-password`;
+
+  let orgToken;
+
+  beforeAll(async () => {
+    orgToken = await getOrgToken(global.organization._id, global.admin._id);
+  });
```

Comment:
Nice catch! We forgot to include this among the global variables. This is actually in `__test__/config/setup.js`. Would've been great if you helped fix this by placing it there so that it is available to all tests. Thanks